### ISSUE TITLE
Black screen after launch DSimenu++ in DSimenu++ launcher

### ISSUE THREAD

**Issue by a00909** [Robz8/DSiMenuPlusPlus#190]
### Your console

Tell us which console do you have:

- [ ]  DS Phat
- [ ]  DS Lite
- [ ]  DSi
- [ ]  DSi XL
- [ ]  Old 3DS
- [ ]  Old 3DS XL
- [ ]  Old 2DS
- [ ]  New 3DS
- [x]  New 3DS XL
- [ ]  New 2DS XL

### CFW and configuration

Tell us which CFW do you have:

DSi:

- [ ]  HiyaCFW
- [ ]  Not using CFW / Unlaunch only
- [ ]  Not using CFW / DSiWarehax

3DS/2DS:

- [x]  Luma3DS
- [ ]  Rei-Six
- [ ]  Other CFW [specify]
- [ ]  Older CFW

### 3DS console version (ignore, if you're using DS/DSi)

Tell us which version do you have:

- [x]  >= 11.4
- [ ]  < 11.4

### DSiMenu++ Launcher version (ignore, if you're using DS/DSi)

This launches DSiMenu++ on 3DS/2DS.

Tell us which versions you are using:

- [x]  Latest release _[2.0.2]_
- [ ]  I've compiled myself _[insert here commit hash]_
- [ ]  Old release _[please specify version]_

### DSiMenu++ version

Tell us which versions you are using:

- [x]  Latest release _[6.0.0]_
- [ ]  I've compiled myself _[insert here commit hash]_
- [ ]  Old release _[please specify version]_

### Expected behaviour

Tell us what should happen
I think it should launch dsimenu++.

### Actual behaviour

Tell us what happens instead
But it didn't. Black screen for long time until I turn off the power.
I turned on the led in rainbow mode and found that it's breathing when it is black screen.
I have tried this: 
https://3ds.hacks.guide/troubleshooting#dsi--ds-functionality-is-broken-after-completing-the-guide
but it still not works
### Steps to reproduce

1. put _nds, BOOT.NDS, cia and dsimenu++_launcher to root of 3ds 
2. install 3 cias via FBI
3. run dsimenu++_launcher
4. select start dsimenu++

**Comment by NightYoshi370** [comment]
Do you have an EmuNAND?

**Comment by a00909** [comment]
Yes. I use Luma to get into an emunand

**Comment by NightYoshi370** [comment]
Did you install the Cias into your system NAND as well?

**Comment by a00909** [comment]
No. I install all cias into emuNand. I had separated sysNand and emuNand.

**Comment by NightYoshi370** [comment]
You'll need to install your CIA's to the sysNAND as well as EmuNAND.

**Comment by a00909** [comment]
I don't know how to install CIAs to sysNand, so today i tried using Boot9strap + luma, and it worked.
Thank you so much.
